### PR TITLE
feat(STONEINTG-560): alerting for IS SLO 2A

### PR DESCRIPTION
**Pull request by MartinBasti**
Adding alerting for snaphot created to integration PLR in static evn created SLO.

### CODE REVIEW

File: rhobs/alerting/data_plane/prometheus.latency_snapshot_to_static_integration_plr_creation_alerts.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
+          Latency from an application snapshot created to integration PLRs in static envs created.
+        description: >
+          Time from Snapshot created to integration PLRs in static envs created has been over
+          5s for more than 10% of requests during the last 5 minutes on cluster
+          {{ $labels.source_cluster }}
+        runbook_url: https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/integration-service/latency_snapshot_to_integration_test_static.md
```
Comment (MartinBasti) [inline rhobs/alerting/data_plane/prometheus.latency_snapshot_to_static_integration_plr_creation_alerts.yaml:33]: Draft because this is not merged yet

Comment (MartinBasti) [reply rhobs/alerting/data_plane/prometheus.latency_snapshot_to_static_integration_plr_creation_alerts.yaml:33]: And also issues with unittests

Comment (yftacherzog) [reply rhobs/alerting/data_plane/prometheus.latency_snapshot_to_static_integration_plr_creation_alerts.yaml:33]: You can add `{{ $value | humanizePercentage }}` to your summary/description to have better view of the query value. That might help with unit tests as well.

Comment (yftacherzog) [reply rhobs/alerting/data_plane/prometheus.latency_snapshot_to_static_integration_plr_creation_alerts.yaml:33]: Note that on some of the input series, you provide 6min-worth of inputs (`a+bx5` == `a a+b a+2b ... a+5b`), while your evaluation time of the rule is at 10 minutes.

File: test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      - series: 'integration_svc_snapshot_created_to_pipelinerun_with_static_env_started_seconds_bucket{le="5", source_cluster="cluster01"}'
+        values: '0+5x5'  # 5 requests took less than 5s
+      - series: 'integration_svc_snapshot_created_to_pipelinerun_with_static_env_started_seconds_bucket{le="+Inf", source_cluster="cluster01"}'
+        values: '0+100x5'  # 100 total occurrences in cluster01
+
+      # Simulating data from Cluster 2 (also crosses the 10% threshold)
+      - series: 'integration_svc_snapshot_created_to_pipelinerun_with_static_env_started_seconds_bucket{le="5", source_cluster="cluster02"}'
+        values: '0+5x10'  # 5 requests took less than 5s
+      - series: 'integration_svc_snapshot_created_to_pipelinerun_with_static_env_started_seconds_bucket{le="+Inf", source_cluster="cluster02"}'
+        values: '0+100x5'  # 100 total occurrences in cluster02
```
Comment (yftacherzog) [inline test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml:51]: The two lines below are interchangeable: 
```
values: '0+5x10'
values: '0 5 10 15 ... 50'
```
This means that on the first series, you give 11 minutes of measurements to the le=5 series while on the second series you give 6 minutes of measurements to the le=inf series.

I'm not sure this is what you're after given the comments.

See reference here: https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/#series

Comment (MartinBasti) [reply test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml]: Thank you very much for helpful comments, I messed a lot.

File: test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+rule_files:
+  - 'prometheus.latency_snapshot_to_static_integration_plr_creation_alerts.yaml'
+
+tests:
+  - interval: 1m
+    input_series:
+      # Simulating data from Cluster 1 (crosses the 10% threshold)
+      - series: 'integration_svc_snapshot_created_to_pipelinerun_with_static_env_started_seconds_bucket{le="5", source_cluster="cluster01"}'
+        values: '0+10x10'  # 10 requests took less than 5s
```
Comment (yftacherzog) [inline test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml:11]: The comment might be a bit misleading. It's 10 requests per minute (same goes for other comments).

Comment (MartinBasti) [reply test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml:11]: It's ready to be merged, but I don't have issues with updating the comments

Comment (MartinBasti) [reply test/promql/tests/data_plane/latency_snapshot_to_static_integration_plr_creation_test.yaml]: updated PTAL

**Review by yftacherzog — APPROVED** [review]

**Comment by MartinBasti** [comment]
Thank you sir for reviews!

**Pull request merged**